ios/Exponent/Kernel/Services/EXKernelLinkingManager.m

```diff
@@ -271,6 +271,7 @@ + (BOOL)_isExpoHostedUrlComponents: (NSURLComponents *)components
   if (components.host) {
     return [components.host isEqualToString:@"exp.host"] ||
       [components.host isEqualToString:@"expo.io"] ||
+      [components.host isEqualToString:@"expo.dev"] ||
```

Comment:
I'm not sure we want this. Projects are not hosted on expo.dev (we are planning to add u.expo.dev).

This file looks like it for forwarding deep links into running projects, which we don't nneed/want to do for expo.dev/expo-go. We want this URL simply to open Expo Go if it's on the device and do nothing.